project/Dependencies.scala

```diff
@@ -71,24 +71,26 @@ object Dependencies {
   private val owlApiV = "5.1.16"
   private val paradiseV = "2.1.1"
   private val pegdownV = "1.6.0"
-  private val postgresV = "42.2.5"
+  // For org.postgresql:postgresql 42.2.6 - 42.2.14:
+  // java.lang.NoSuchFieldException: m_mesgParts in KeyValueSpec "fail if one of the inserts fails"
+  private val postgresV = "42.2.5" // scala-steward:off
   private val rdf4jV = "2.4.2"
   private val refinedV = "0.9.8"
-  private val rhinoV = "1.7.10"
-  private val scalaGraphV = "1.12.5"
+  private val rhinoV = "1.7.12"
+  private val scalaGraphV = "1.13.1"
   private val scalaLoggingV = "3.9.2"
   private val scalaPoolV = "0.4.3"
   private val scalacheckV = "1.14.0"
-  private val scalacticV = "3.0.5"
+  private val scalacticV = "3.0.8"
   private val scalameterV = "0.10.1"
-  private val scalamockV = "4.1.0"
+  private val scalamockV = "4.4.0"
   private val scalatestV = "3.0.5"
-  private val scalazV = "7.2.27"
+  private val scalazV = "7.2.30"
```

Comment:
TIL we still import scalaz!??

Reply:
and it's still being updated, it's actually up to 7.4 now